### PR TITLE
kinder/hack: add verify and update scripts

### PR DESCRIPTION
**Pull request by neolit123**
/assign @fabriziopandini 
/priority important-longterm
/kind feature

please review one commit at a time.
i will setup the presubmit CI for this repo next week.

### CODE REVIEW

File: kinder/hack/update-deps.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    -not -iname "CONTRIBUTORS*" \
+    -not -iname "COPYING*" \
+    -not -iname "LICENSE*" \
+    -not -iname "NOTICE*" \
+    -exec rm '{}' \;
+}
+
+export GO111MODULE="on"
+go mod tidy
+# TODO enable in kinder?
```
Comment (fabriziopandini) [inline kinder/hack/update-deps.sh]: Unless necessary, I think we should wait until kinder gets more stable (end of this cycle) before starting to vendor, so we can change without getting PR big.

File: kinder/hack/update-deps.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# go mod vendor
+# prune-vendor
+
+# TODO(bentheelder): re-enable once (if?) we're using docker client
```
Comment (fabriziopandini) [inline kinder/hack/update-deps.sh]: I don't think this apply to kinder

File: kinder/hack/update-deps.sh (hunk 1 above)
Comment (neolit123) [reply kinder/hack/update-deps.sh]: ack.

File: kinder/hack/update-deps.sh (hunk 2 above)
Comment (neolit123) [reply kinder/hack/update-deps.sh]: updated.

**Comment by neolit123** [comment]
/hold
looks like there is a bug in update-deps.sh.

one concern here is that with the k8s publishing bot the deps update daily for api-machinery.

**Pull request merged**